### ISSUE TITLE
Get back to "decision pending" hackEDU applicants

### ISSUE THREAD

**Issue by zachlatta** [hackedu/meta#145]
I've created a working column in the Applicants sheet in https://clubs.hackedu.us to track this.

Legend:

- `WAITLIST` :: Put at bottom of waitlist
- `WAITLIST ACCEPT` :: Put them on the waitlist to be a hackEDU club starting next semester. Essentially a deferred acceptance.
- `WAITLIST INTERVIEW` :: Say that you'll follow up with them next semester. Set a time to reach out and schedule a call where you can interview them again.
- `ACCEPT` :: Accept them for this semester.

If you write the copy, I can send it out. This needs to be done before I fly out on Wednesday morning.

**Comment by hellyeah** [comment]
Writing this email now.

**Comment by hellyeah** [comment]
Subject: hackEDU Club Decision
Alt: hackEDU Waitlist

"We would like to invite you to be one of the first clubs to pilot the hackEDU playbook next semester. 

Though we won't be accepting new hackEDU clubs this semester, we've created a Labs program of 12 HS hack club leaders who are developing the playbook for building an amazing hack club. You can follow their progress here: https://github.com/hackedu/hackedu

We'll never stop iterating on the playbook -- feel free to begin using it now and submitting feedback right away through github issues: https://github.com/hackedu/hackedu/issues

Look forward to an email from us at the end of the semester. We'll be expanding quickly and want hackEDU to help you build the best hack club in the world.

-Zach"

**Comment by zachlatta** [comment]
Awesome! Thank you.

Can you also write one for accepting them this semester?

**Comment by hellyeah** [comment]
Subject: Welcome to hackEDU

"We would like to invite your club to hackEDU this semester.

We're currently focused on the hackEDU Labs program, a cohort of 12 club leaders who are developing the (Hack Club Playbook)[https://github.com/hackedu/hackedu], so we won't be devoting as much time to hackEDU clubs until the following semester.

That said, we think you'll add value to the hackEDU Leaders community immediately so we wanted to invite you to join right away: bit.ly/hackEDULeaders

When you join the Facebook Group, make a post introducing yourself and bringing up a challenge you anticipate facing with your club this year.

**Comment by zachlatta** [comment]
Perfect, thank you.

On Tue, Sep 15, 2015 at 4:12 PM, David Fontenot <notifications@github.com>
wrote:

> Subject: Welcome to hackEDU
>
> "We would like to invite your club to hackEDU this semester.
>
> We're currently focused on the hackEDU Labs program, a cohort of 12 club
> leaders who are developing the (Hack Club Playbook)[
> https://github.com/hackedu/hackedu], so we won't be devoting as much time
> to hackEDU clubs until the following semester.
>
> That said, we think you'll add value to the hackEDU Leaders community
> immediately so we wanted to invite you to join right away:
> bit.ly/hackEDULeaders
>
> When you join the Facebook Group, make a post introducing yourself and
> bringing up a challenge you anticipate facing with your club this year.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/meta/issues/145#issuecomment-140577868>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by hellyeah** [comment]
@zachlatta moving forward if you ever need a quick draft of something written just ping me. It won't be perfect, but I'm getting in a much better productivity flow of shipping drafts and can likely save you a lot of time getting things like this done.

**Comment by zachlatta** [comment]
That sounds perfect. I'll certainly take you up on this.

**Comment by jonleung** [comment]
@zachlatta did you send this out yet?

**Comment by zachlatta** [comment]
About to.

On Wednesday, September 16, 2015, Jonathan Leung <notifications@github.com>
wrote:

> @zachlatta <https://github.com/zachlatta> did you send this out yet?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/meta/issues/145#issuecomment-140788929>.
>


-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by hellyeah** [comment]
Subject: hackEDU Club Decision
Alt: hackEDU Waitlist

"We decided to focus this semester on the hackEDU Labs program where 12 high school club leaders will be developing the definitive playbook for building a coding club from scratch.

Though we won't be accepting your club to hackEDU this semester, you can follow progress on the playbook here: https://github.com/hackedu/hackedu

We intentionally do everything openly on GitHub so that you have access to the playbook and other hackEDU materials as soon as we do -- we encourage you to contribute.

Feel free to start implementing the playbook in your club right away. No need to ask us for permission to use any of the hackEDU workshops in your club either.

Look forward to an email from us in December. We'll be expanding hackEDU to include more high schools next semester and would love to talk to you about joining the hackEDU community then.

-Zach"

**Comment by MaxWofford** [comment]
@hellyeah I'm currently adding to the clubs list. I'll add a comment here when I've finished.

**Comment by jonleung** [comment]
@hellyeah, made some minor revisions. I think it's good to send.

Subject: hackEDU Club Decision
Alt: hackEDU Waitlist

This semester, we decided to focus on the hackEDU Labs program where 12 high school club leaders will be developing a playbook for building a coding club from scratch.

Though we won't be accepting your club to hackEDU this semester, you can follow progress on the playbook here: https://github.com/hackedu/hackedu

We intentionally do everything openly on GitHub so that you have access to the playbook and other hackEDU materials as soon as we do, and we encourage you to contribute.

Feel free to start implementing the playbook in your club right away. No need to ask us for permission to use any of the hackEDU workshops in your club either.

Look for email from us in December, when we'll be expanding hackEDU to include more high schools next semester. We would love to talk to you about joining the hackEDU community then!

-Zach"

**Comment by hellyeah** [comment]
Did not email: 

Joshua Ng 14ngk3@rchk.edu.hk
Nicholas Salazar koipasta3@gmail.com

**Comment by MaxWofford** [comment]
Added all the applicants I received over summer.

**Comment by hellyeah** [comment]
@zachlatta do we want to send this email to WAITLIST_ACCEPT?
